instance/models/openedx_appserver.py

```diff
@@ -163,6 +167,17 @@ class Meta:
         )
     )
 
+    def get_configuration_playbook_name(self):
```

Comment:
@giovannicimolin, wouldn't it be cleaner to make this a property?

Reply:
Yes, and done.